src/me/nikl/cookieclicker/updates/factory/ChineseWorkers.java

```diff
@@ -0,0 +1,49 @@
+package me.nikl.cookieclicker.updates.factory;
+
+import me.nikl.cookieclicker.Game;
+import me.nikl.gamebox.util.NumberUtil;
+import me.nikl.cookieclicker.buildings.Buildings;
+import me.nikl.cookieclicker.updates.Upgrade;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.material.MaterialData;
+
+import java.util.ArrayList;
+
+/**
+ * Created by Niklas on 09.07.2017.
+ *
+ */
+public class ChineseWorkers extends Upgrade{
+
+    public SturdierConveyorBelts(Game game) {
+        super(game, 13);
```

Comment:
Sorry about the class constructor, was just a quick idea written down in a simple text editor ;)
Feel free to adapt the costs as you think =)

I will come up with more updates soon ^^